ds-prep-python-variables/main.py

```diff
@@ -0,0 +1,34 @@
+first_name = "Jenna"
```

Comment:
That's not single-quotes...

---

![image](https://user-images.githubusercontent.com/16143771/146484113-932d46c9-08cc-4580-b3db-64ea4a16e300.png)